README.md

```diff
@@ -1291,6 +1313,17 @@ They are not guides or best practices for configuring a cluster.
     - linux-system-roles.ha_cluster
 ```
 
+### Configuring firewall and selinux using each role
```

Comment:
I think that instead of having this example, the role must set `ha_cluster_manage_firewall: true and `ha_cluster_manage_selinux: true` in all other examples alongside required variables. Because now when these vars are set to `false` by default, other examples won't work properly. What do you think @nhosoi ?

Reply:
Ah, I see... Thank you for the clarification, @spetrosi. It should be better as examples could be used in the copied and pasted manner... I guess we should even emphasize it more with stronger recommendation to set them `true`.

@spetrosi, are you willing to modify the README file or shall I work on it?

Reply:
I guess that is required in all roles where you `_manage_firewall` and `_manage_selinux` are added. I would prefer you to do this because you added them in the first place, but I can too if you have a lot on your plate.

Reply:
@spetrosi, I've submitted https://github.com/linux-system-roles/ha_cluster/pull/70 for the examples.
I did not add the manage firewall/selinux variables in each example in README.md, but did so in the playbooks in the `examples` directory. And I'm trying to convince customers to set the variables to true when they create their playbook.

Could you please review it and share your thoughts? (Needless to say :), please feel free to rewriting the sentences/examples in README. Thanks!